src/components/views/messages/TileErrorBoundary.tsx

```diff
@@ -92,8 +92,9 @@ export default class TileErrorBoundary extends React.Component<IProps, IState> {
                 <div className="mx_EventTile_line">
                     <span>
                         { _t("Can't load this message") }
-                        { mxEvent && ` (${mxEvent.getType()})` }
+                        { mxEvent && ` (${mxEvent.getType()}) ` }
                         { submitLogsButton }
+                        { ` ` }
```

Comment:
It would probably be better to use `&nbsp;`; see https://www.w3schools.com/html/html_entities.asp